pytests/conftest.py

```diff
@@ -101,36 +101,44 @@ def __init__(self, cli_args=None):
             self.config.update(cli_args)
         self.config['distribution'] = os.environ.get('DIST', 'photon')
         script = os.path.join(self.config['test_path'], 'repo/setup-repo.sh')
-        self.run([ 'sh', script, self.config['repo_path'] ])
+        self.run(['sh', script, self.config['repo_path']])
         self.tdnf_config = configparser.ConfigParser()
-        self.tdnf_config.read(os.path.join(self.config['repo_path'], 'tdnf.conf'))
+        self.tdnf_config.read(os.path.join(self.config['repo_path'],
+                                           'tdnf.conf'))
 
         # check execution environment and enable valgrind if suitable
         self.check_valgrind()
 
     def version_str_to_int(self, version):
         version_parts = version.split('.')
-        return version_parts[0] * 1000 + version_parts[1] * 100 + version_parts[2]
+        version = version_parts[0] * 1000
+        version += version_parts[1] * 100
+        version += version_parts[2]
+        return version
 
     def check_valgrind(self):
         self.config['valgrind_enabled'] = False
-        valgrind = distutils.spawn.find_executable('valgrind')
+        valgrind = shutil.which('valgrind')
         if not valgrind:
             self.config['valgrind_disabled_reason'] = 'valgrind not found'
             return
 
-        stream = os.popen(valgrind + ' --version') #nosec
+        stream = os.popen(valgrind + ' --version')  # nosec
         valgrind_version = stream.read()
         if not valgrind_version:
-            self.config['valgrind_disabled_reason'] = 'Unable to ascertain valgrind version'
+            reason = 'Unable to ascertain valgrind version'
+            self.config['valgrind_disabled_reason'] = reason
             return
 
         # valgrind versions less than minimum required might not support
         # all the options specified or might have known errors.
-        current_version = self.version_str_to_int(valgrind_version.replace('valgrind-', ''))
-        minimum_version = self.version_str_to_int(self.config['valgrind_minimum_version'])
+        current_version = valgrind_version.replace('valgrind-', '')
+        current_version = self.version_str_to_int(current_version)
+        minimum_version = self.config['valgrind_minimum_version']
+        minimum_version = self.version_str_to_int(minimum_version)
```

Comment:
You add extra steps here. This is not an improvement.

Reply:
Same here, miserable effort towards keeping line under 80 columns.

Reply:
It's still there though. Please revert. You an split a command over multiple lines without extra steps.